.circleci/config.yml

```diff
@@ -0,0 +1,74 @@
+version: 2.1
+
+jobs:
+    build_docker_environment:
+        parameters:
+            pytorch_channel:
+                type: string
+                default: "pytorch"
+        machine:
+            image: ubuntu-1604:201903-01
```

Comment:
https://github.com/pytorch/serve/pull/431/files#r435594431
seems to use Ubuntu 18.04 docker image and build on top.
I'm not sure what the `image:` tag is for or what the naming convention used?
`ubuntu-1604:201903-01`

Reply:
These are related to circleci's base image and although they look similar do not relate to docker images.

see https://circleci.com/docs/2.0/configuration-reference/#available-machine-images